### ISSUE TITLE
Loading MoViNet saved_model in tensorflow 2.7 throws FileNotFoundError: Op type not registered 'DisableCopyOnRead' in binary running

### ISSUE THREAD

**Issue by hammad-ali1** [tensorflow/tensorflow#62229]
### Issue type

Support

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

binary

### TensorFlow version

tf 2.7

### Custom code

Yes

### OS platform and distribution

Linux Ubuntu 18.04

### Python version

3.6

### GPU model and memory

Jetson Nano Maxwell Architecture GPU - 4GB

### Current behavior?

I am trying to load an exported MoViNet saved model in tensorflow version 2.7. The model loads fine in tensorflow version 2.10. However, I need to run the model on Nvidia's Jetson nano which does not support tensorflow version greater than 2.7. Loading the model in tf 2.7 gives me the error `Op type not registered 'DisableCopyOnRead'`. This issue might be due to version compatibility  since I exported the model using tensorflow 2.10 and tf-models-official. For exporting model I used :
[this notebook's code](https://github.com/tensorflow/models/blob/master/official/projects/movinet/movinet_streaming_model_training_and_inference.ipynb).
I tried to export the model using tf 2.7 but tf-models-official does not work with tf<2.10. 

Are there any workarounds by converting the model in some other format so that I can run it on my Jetson Nano's tensorflow version 2.7.
Any help would be greatly appreciated. Much thanks.

### Standalone code to reproduce the issue

```shell
import tensorflow as tf

model = tf.saved_model.load("path/to/saved_model")
```

### Relevant log output

```shell
KeyError                                  Traceback (most recent call last)
[c:\Users\Hammad\anaconda3\envs\temp\lib\site-packages\tensorflow\python\framework\ops.py](file:///C:/Users/Hammad/anaconda3/envs/temp/lib/site-packages/tensorflow/python/framework/ops.py) in _get_op_def(self, type)
   4097     try:
-> 4098       return self._op_def_cache[type]
   4099     except KeyError:

KeyError: 'DisableCopyOnRead'

During handling of the above exception, another exception occurred:

NotFoundError                             Traceback (most recent call last)
[c:\Users\Hammad\anaconda3\envs\temp\lib\site-packages\tensorflow\python\saved_model\load.py](file:///C:/Users/Hammad/anaconda3/envs/temp/lib/site-packages/tensorflow/python/saved_model/load.py) in load_internal(export_dir, tags, options, loader_cls, filters)
    938         loader = loader_cls(object_graph_proto, saved_model_proto, export_dir,
--> 939                             ckpt_options, options, filters)
    940       except errors.NotFoundError as err:

[c:\Users\Hammad\anaconda3\envs\temp\lib\site-packages\tensorflow\python\saved_model\load.py](file:///C:/Users/Hammad/anaconda3/envs/temp/lib/site-packages/tensorflow/python/saved_model/load.py) in __init__(self, object_graph_proto, saved_model_proto, export_dir, ckpt_options, save_options, filters)
    138         function_deserialization.load_function_def_library(
--> 139             meta_graph.graph_def.library, wrapper_function=_WrapperFunction))
    140     self._checkpoint_options = ckpt_options

[c:\Users\Hammad\anaconda3\envs\temp\lib\site-packages\tensorflow\python\saved_model\function_deserialization.py](file:///C:/Users/Hammad/anaconda3/envs/temp/lib/site-packages/tensorflow/python/saved_model/function_deserialization.py) in load_function_def_library(library, load_shared_name_suffix, wrapper_function)
    387     with graph.as_default():
--> 388       func_graph = function_def_lib.function_def_to_graph(copy)
...
    943             "from the computational device. Consider setting the "
    944             "`experimental_io_device` option in `tf.saved_model.LoadOptions` "

FileNotFoundError: Op type not registered 'DisableCopyOnRead' in binary running on LAPTOP-V4DO1PQN. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.
 You may be trying to load on a different device from the computational device. Consider setting the `experimental_io_device` option in `tf.saved_model.LoadOptions` to the io_device such as '/job:localhost'.
```

**Comment by hammad-ali1** [comment]
@yuanliangzhe @yeqingli Can you guys please give me any pointers on how to build and export the MoViNet model in tensorflow 2.7 without relying on  tf-models-official package. Or alternatively, load an already exported saved_model in tf 2.7. Note that, the tflite version of the model loads fine. It is the saved_model format that's causing the issue.

**Comment by hammad-ali1** [comment]
The issue was resolved by building the model int tensorflow version 2.9. When tf-models-official package was installed it updated the tensorflow version to 2.14. Somehow, MoViNet model built in this version was not able to load in tf2.7. 
I downgraded tensorflow to the least possible version that was compatible with tf-models-official and then build the model using saved weights. After which I exported the model in saved_model format and it worked with tf2.7.
